test/test_integration.py

```diff
@@ -496,3 +496,27 @@ def update_graph(n_clicks):
         button.click()
         time.sleep(2)
         self.snapshot('candlestick - 2 click')
+
+    def test_latex(self):
+        app = dash.Dash(__name__)
+        app.layout = html.Div([
+
+            html.Label('Graph'),
+            dcc.Graph(
+                id='graph',
+                figure={
+                    'data': [{
+                        'x': [1, 2, 3, 4, 5],
+                        'y': [1, 3, 5, 9, 13]
+                    }],
+                    'layout': {
+                        'title': 'Graph of \(y = \frac{x^2 - 2}{4}\)'
+                    }
+                }
+            ),
+        ])
+        self.startServer(app=app)
+
+        graph = self.wait_for_element_by_css_selector('#graph')
```

Comment:
Here's an idea - let's "wait for" the actual graph to be rendered rather than just the container. Let's update this CSS selector to be:
```
'#graph .svg-container'
```